Sources/RealDeviceMapLib/Webhook/WebHookRequestHandler.swift

```diff
@@ -269,14 +270,20 @@ public class WebHookRequestHandler {
                             hasArQuestReq ??
                             getArQuestMode(device: uuid, timestamp: timestamp)
                         let challengeQuest = fsr.challengeQuest
-                        if challengeQuest.quest.questType == .questGeotargetedArScan && uuid != nil {
-                            questArActualMap.setValue(key: uuid!, value: true, time: timestamp)
+                        // Ignore AR quests so they get rescanned if they were the first quest a scanner would hold onto
```

Comment:
@Kneckter

Reply:
Setting it to `true` would run like RDM runs now. Setting it to `false` would be in-line with the default quest instance since it defaults to `normal` mode (that default is in dashboard_instance_add.mustache and InstanceController.swift). I don't feel strongly about it either way since I think people should read about new configs when they are merged. 

Setting it to `true` would be the more backwards compatible since the blocking wasn't provided in the past so I'll push that change to the json.